images/runtime/ruby/2.5/buster.Dockerfile

```diff
@@ -20,9 +20,10 @@ ENV RUBY_VERSION 2.5.8
 RUN ${IMAGES_DIR}/installPlatform.sh ruby $RUBY_VERSION --dir /opt/ruby/$RUBY_VERSION --links false
 RUN set -ex \
  && cd /opt/ruby/ \
- && ln -s 2.5.8 2.5
+ && ln -s 2.5.8 2.5 \
+ && ln -s 2.5 2
 
-ENV PATH="/opt/ruby/2.5/bin:${PATH}"
+ENV PATH="/opt/ruby/2/bin:${PATH}"
```

Comment:
nit: just curious about the need for these changes in the Ruby runtimes.

Reply:
These changes were added automatically when I [fixed the generateDockerfiles script for ruby](https://github.com/microsoft/Oryx/pull/1529/files#diff-f379ee5e93788de18e787b3f0dea9986fc1c78276ff03ebf7d73188a470b9eb5) in order to test my changes. Turns out that the generate dockerfiles script wasn't actually going over the versions directories due to a small typo.

Reply:
Got it -- if there were changes added as a result of running a script in our repository, then I think we _should_ be safe to keep them in this PR.